### PR TITLE
Microsoft Entra Rebrand

### PR DESCRIPTION
**Pull request by DidunAyodeji**
Second try at updating this repo with a new script.

Updating all Azure AD terms to the Microsoft Entra ID terms in all md files.

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
The updated changes to Microsoft Entra ID terminology make sense.

## Other Information
Here's a link to the Rebrand guidelines. Note that the script doesn't have all the entries in the glossary that mine has. Let me know if you'd like to see my version of the script. https://review.learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad?branch=main

### CODE REVIEW

File: 1. Desktop app calls Web API/README-incremental.md [hunk 1]
```diff
@@ -337,7 +337,7 @@ See [2. Web API now calls Microsoft Graph](../2.%20Web%20API%20now%20calls%20Mic
 
 Use [Stack Overflow](http://stackoverflow.com/questions/tagged/msal) to get support from the community.
 Ask your questions on Stack Overflow first and browse existing issues to see if someone has asked your question before.
-Make sure that your questions or comments are tagged with [`azure-active-directory` `msal` `dotnet`].
+Make sure that your questions or comments are tagged with [`microsoft-entra-id` `msal` `dotnet`].
```
Comment (jmprieur) [inline 1. Desktop app calls Web API/README-incremental.md:340]: Was the "azure-active-directory" tag removed in Stack overflow. It seems to me that it enables people to troubleshoot themselves. I would keep it here (find historical issues?)

File: 1. Desktop app calls Web API/README-incremental.md [hunk 2]
```diff
@@ -359,4 +359,4 @@ For more information, visit the following links:
 
 - To learn more about ASP.NET Core Web APIs: see [Introduction to Identity on ASP.NET Core](https://docs.microsoft.com/aspnet/core/security/authentication/identity?view=aspnetcore-2.1&tabs=visual-studio%2Caspnetcore2x) and also:
   - [AuthenticationBuilder](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.authentication.authenticationbuilder?view=aspnetcore-2.0)
-  - [Azure Active Directory with ASP.NET Core](https://docs.microsoft.com/aspnet/core/security/authentication/azure-active-directory/?view=aspnetcore-2.1)
+  - [Microsoft Entra ID with ASP.NET Core](https://docs.microsoft.com/aspnet/core/security/authentication/azure-active-directory/?view=aspnetcore-2.1)
```
Comment (jmprieur) [inline 1. Desktop app calls Web API/README-incremental.md:362]:
```suggestion
  - [Microsoft Entra ID with ASP.NET Core](https://docs.microsoft.com/aspnet/core/security/authentication/azure-active-directory)
```

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @DidunAyodeji

I left 2 suggestions which would be great to address.
Thanks!

File: 1. Desktop app calls Web API/README-incremental.md (hunk 1 above)
Comment (DidunAyodeji) [reply 1. Desktop app calls Web API/README-incremental.md:340]: Great catch. I have added this entry to the post-transform section of the script to correct this issue.` @{ Key = '`microsoft-entra-id`'; Value = '`azure-active-directory`' }`

I will run the updated script on this branch to address this issue everywhere and hopefully will prevent this issue in all the other repos that will be updated.

Comment (DidunAyodeji) [reply 1. Desktop app calls Web API/README-incremental.md:340]: thank you!

File: 1. Desktop app calls Web API/README-incremental.md (hunk 2 above)
Comment (DidunAyodeji) [reply 1. Desktop app calls Web API/README-incremental.md:362]: Thank you! Committing your suggestion.

File: 1. Desktop app calls Web API/README-incremental.md (hunk 1 above)
Comment (DidunAyodeji) [reply 1. Desktop app calls Web API/README-incremental.md]: This readme file has been updated with the correct tag now. Thank you again.

**Pull request merged**